src/heliclockter/__init__.py

```diff
@@ -255,6 +255,15 @@ def assert_aware_datetime(dt: _datetime.datetime) -> None:
         """
         assert dt.tzinfo is not None and dt.tzinfo.utcoffset(dt) is not None
 
+    def astimezone(self, tz: _datetime.tzinfo | None = None) -> datetime_tz:
+        """
+        Return a datetime_tz object with the same datetime data but in the specified timezone.
+        Uses local timezone if no timezone is provided.
+        """
+        if tz is None:
+            tz = tz_local
+        return datetime_tz.fromtimestamp(self.timestamp(), tz=tz)
```

Comment:
Nice, I like this better than returning a `datetime_utc` with an offset like we briefly discussed 👍🏼